en/PlanView/PlanView.md

```diff
@@ -11,33 +11,39 @@ The image above shows a simple mission which starts with a takeoff, flies throug
 The steps to creating a missions are:
 
 1. Change to Plan View
-2. Add commands to the mission and edit as needed
-3. Send the mission to the vehicle
+2. Add waypoints to the mission and edit as needed
+3. Upload: Send the mission to the vehicle
 4. Change to Fly View and fly your mission
 
 ## Plan Tools
 On the left edge of the screen you will see the Plan Tools. The order of tools from top to bottom is:
 
-* Add Commands
-* [Survey](Survey.md)
+* Add waypoints
```

Comment:
As above, agree terminology and stick to it.

Reply:
The new QGC lists texts below the icons where it only says "Waypoints". Should I remove this section, mark icons in red on QGC image above and in this section refer to the marked icons?

Reply:
Essentially  what you are trying to do here is make it clear what the tools are from their icons. I think that would be better with a labeled image, so yes.